iceoryx_platform/BUILD.bazel

```diff
@@ -64,37 +65,61 @@ configure_version(
     version_from = "//:VERSION",
 )
 
+configure_file(
+    name = "platform_settings_hpp",
+    src = select({
+        ":linux": "linux/cmake/platform_settings.hpp.in",
+        ":mac": "mac/cmake/platform_settings.hpp.in",
+        ":qnx": "qnx/cmake/platform_settings.hpp.in",
+        ":unix": "unix/cmake/platform_settings.hpp.in",
+        ":win": "win/cmake/platform_settings.hpp.in",
```

Comment:
Aren't we forgetting the FreeRTOS platform here. Ouh I see a dark cloud on the horizon. FreeRTOS will most likely no longer work and we need urgently a CI test for it.

Reply:
Yeah, FreeRTOS was never added to bazel. We also only run linux with bazel on the CI so there are chances it is also broken for other targets but since there are no bug reports it either works or nobody is using it